Mobile/starter_project_matador/lib/features/profile page/presentation/widget/header_widget.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+
+class Profile_Text extends StatelessWidget {
+  const Profile_Text({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const Padding(
+      padding: EdgeInsets.fromLTRB(30, 15, 0, 0),
+      child: Text(
+        "Profile",
+        style: TextStyle(
```

Comment:
Define All the styles in the core/utils directory and reuse them whenever needed.